zombies.js

```diff
@@ -208,6 +237,136 @@
  * @return {string/boolean}   Weapon name or false if nothing is equipped.
  */
 
+// class Player {
+//   constructor (name, health, strength, speed){
+//     this.name = name;
+//     this.health = health;
+//     this.strength = strength;
+//     this.speed = speed;
+//     this.isAlive = true;
+//     this.equipped = false;
+//     this._pack = [];
+//     this._maxHealth = health;
+//   }
+
+function Player (name, health, strength, speed){
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+  let pack = [];
+  this.getPack = function (){
+    return pack;
+  };
+  this.getMaxHealth = function (){
+    return health;
```

Comment:
I think you're returning the wrong variable here. Also where is the _maxHealth variable declared?